beeline/pom.xml

```diff
@@ -99,6 +99,20 @@
       <version>${project.version}</version>
       <classifier>tests</classifier>
       <scope>test</scope>
+      <exclusions>
```

Comment:
Looks good as I no longer see the test failure 👍 . BTW do you know why we need to exclude calcite here but not in other places where `hive-exec` is also used?

Reply:
For test scope, I think the test-jar won't include shaded dependencies. Then un-shaded dependencies will be pull. So I think we need to exclude them.

If there are other places using hive-exec in test scope, I think they might also need to exclude these.

Reply:
Ah I see. I think there are a few other places with similar pattern, such as `hive-webhcat-java-client`. Should we exclude in all those places?

Reply:
Yea, I think so. I will update this later.

Reply:
I updated all hive-exec test-jar usage in all pom.xml files.

Reply:
+1. Can you create a JIRA for this? I'm not sure if master branch also have this issue.

Reply:
Created a JIRA. Updated the title and description.